src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerPanel.java

```diff
@@ -703,6 +705,7 @@ private void initializePanelItems() {
 		_weightedOptionGroup.add(_weightedAdditive);
 		_weightedOptionGroup.add(_weightedProbabilities);
 
+		_includePathScoreTiesOption = new JCheckBox("<html>Include paths above k if equal path score</html>");
```

Comment:
Change it to "Include more than k paths if they have the same path length/score as the k<sup>th</sup> path"